### PR TITLE
[ru] Translate mutual-transport-layer-security.md

### PR DESCRIPTION
**Pull request by Squoresha**
### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by Squoresha** [comment]
Please, check it :)

File: content/ru/mutual-transport-layer-security.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Протокол взаимной защиты транспортного уровня (mTLS)
+status: Completed
+category: Concept
+tags: ["security", "networking", ""]
+---
+
+Протокол взаимной защиты транспортного уровня — это метод, используемый для аутентификации и шифрования сообщений, передаваемых между двумя [сервисами](/service/). 
+Протокол взаимной защиты транспортного уровня — это расширение стандартного [Протокола защиты транспортного уровня](/transport-layer-security/) (TLS), 
+но вместо проверки подлинности только сервера, осуществляется взаимная аутентификация.
```
Comment (shurup) [inline content/ru/mutual-transport-layer-security.md:10]:
```suggestion
но вместо проверки подлинности только одной стороны осуществляется валидация обеих.
```

File: content/ru/mutual-transport-layer-security.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Какую проблему решает
+
+[Микросервисы](/microservices-architecture/) осуществляют взаимодействие по сети и, как и ваша wifi сеть, канал коммуникации может быть взломан.
```
Comment (shurup) [inline content/ru/mutual-transport-layer-security.md:14]:
```suggestion
[Микросервисы](/microservices-architecture/) взаимодействуют по сети, но этот канал коммуникации может быть взломан аналогично тому, как это происходит с Wi-Fi.
```

File: content/ru/mutual-transport-layer-security.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+mTLS гарантирует, что никто не сможет прослушивать канал коммуникации и осуществить несанкционированный доступ.
```
Comment (shurup) [inline content/ru/mutual-transport-layer-security.md:15]:
```suggestion
mTLS гарантирует, что неавторизованная сторона не сможет прослушивать канал коммуникации и выполнять несанкционированные действия. 
```

File: content/ru/mutual-transport-layer-security.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+но вместо проверки подлинности только сервера, осуществляется взаимная аутентификация.
+
+## Какую проблему решает
+
+[Микросервисы](/microservices-architecture/) осуществляют взаимодействие по сети и, как и ваша wifi сеть, канал коммуникации может быть взломан.
+mTLS гарантирует, что никто не сможет прослушивать канал коммуникации и осуществить несанкционированный доступ. 
+
+## Как именно решает проблему
+
+mTLS гарантирует, что трафик безопасен и аутентифицирован в обоих направления между клиентом и сервером,
```
Comment (shurup) [inline content/ru/mutual-transport-layer-security.md:19]:
```suggestion
mTLS гарантирует безопасность и аутентичность для трафика в обоих направлениях между клиентом и сервером, 
```

File: content/ru/mutual-transport-layer-security.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+## Какую проблему решает
+
+[Микросервисы](/microservices-architecture/) осуществляют взаимодействие по сети и, как и ваша wifi сеть, канал коммуникации может быть взломан.
+mTLS гарантирует, что никто не сможет прослушивать канал коммуникации и осуществить несанкционированный доступ. 
+
+## Как именно решает проблему
+
+mTLS гарантирует, что трафик безопасен и аутентифицирован в обоих направления между клиентом и сервером, 
+обеспечивая дополнительный уровень безопасности для пользователей, осуществляющих вход в сеть или приложение.
+Он также проверяет соединения с клиентскими устройствами, которые не требуют входа в систему, например, такими как Интернет вещей (IoT).
```
Comment (shurup) [inline content/ru/mutual-transport-layer-security.md:21]:
```suggestion
Он также проверяет соединения с клиентскими устройствами, которые не требуют входа в систему, например, относящимся к категории Интернета вещей (IoT).
```

File: content/ru/mutual-transport-layer-security.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+mTLS может предотвратить сетевые атаки, спуфинг-атаки, атаки способом подстановки учетных данных, взлом пароля путем перебора и другие попытки несанкционированного доступа.
```
Comment (shurup) [inline content/ru/mutual-transport-layer-security.md:22]: I just discovered an exciting [discussion](https://www.reddit.com/r/cybersecurity/comments/ueb5ou/a_question_regarding_mitm_terminology/) regarding on-path vs MiTM attacks…
```suggestion
mTLS может предотвратить атаки посредника (on-path), спуфинг-атаки, атаки с подстановкой учетных данных, атаки путем полного перебора (brute force) и другие. 
```

**Review by shurup — COMMENTED** [review]
Thank you for your ongoing efforts! Here are minor suggestions from me.

**Review by shurup — APPROVED** [review]
Since all my suggestions were accepted, LGTM! Let's wait for @kirkonru now.

**Review by kirkonru — APPROVED** [review]
Thanks a lot for your contribution! LGTM.

**Pull request merged**